### ISSUE TITLE
Add info about glow and the spotted flag etections. 

### ISSUE THREAD

**Issue by dumbasPL** [danielkrupinski/cs2-anticheat#1]
the game has checks against forcing the spotted flag (aka what simple radar hacks use) and also things related to glow. If I don't forget I'll post more info later.

**Comment by dumbasPL** [comment]
on a side note, the new entity system checks seem to target entity VTables iirc. They use the local player controller, local player pawn and loop over the entity list as well.

**Comment by danielkrupinski** [comment]
I wanted to focus first on detections added in CS2, but those from CS:GO are also worth mentioning as they're still getting improvements (for instance DllMain() of client.dll now reads bytes of NtOpenFile and LoadLibraryExW)

**Comment by fL1pSt3r** [comment]
> on a side note, the new entity system checks seem to target entity VTables iirc. They use the local player controller, local player pawn and loop over the entity list as well.

could you please share infromation about these checks?

**Comment by dumbasPL** [comment]
> > on a side note, the new entity system checks seem to target entity VTables iirc. They use the local player controller, local player pawn and loop over the entity list as well.
> 
> could you please share infromation about these checks?

If I recall correctly all it did was grab vtables similar to how the global vtables are checked but this time it's using your local player controller, local pawn, and all/some (I don't remember) entities from the entity list. 
TLDR: don't VMT hook entities or get creative in the way you hook (hook somewhere further up/down the call chain) **OR** hook sending `CUserMessage`'s and spoof the data to the original values (remember to re-calculate the CRC)
